apps/meteor/app/lib/server/methods/saveSettings.ts

```diff
@@ -83,6 +83,12 @@ Meteor.methods<ServerMethods>({
 							break;
 						case 'int':
 							check(value, Number);
```

Comment:
shouldn't this fail if value is an empty string?

Reply:
but I am still curious, how can "value" be NaN for an empty string, after getting serialized.

Reply:
ok I see it, client doesn't just serialize as is.

according to ddp code, while not possible through UI, but is if somebody manually sends Infinity. 

https://github.com/meteor/meteor/blob/7e2abe9464c215aace47d20f737e59243a819871/packages/ejson/ejson.js#L145-L155

suggest using Number.isFinite instead, wdyt @MarcosSpessatto

Reply:
@KevLehman not sure if this is resolved yet 😬

Reply:
I was purposefuly ignoring it :trollface: 

Being serious, that's a possibility 🤔 not sure if it's something that common (to have someone sending `infinity` on the endpoints, but guess it could happen if they have some automated way of doing things.

Bet it would hurt nobody to change it @MartinSchoeler (and it should work the same)

Also, you can add a new test case for sending Infinity or -Infinity to the endpoint

Reply:
Couldn't we use `isInteger`? This way floats, `NaN`, `Infinity` and `-Infinity` would be avoided :thinking:

Reply:
didn't know isInteger is a thing. came to know about isFinite even couple days ago. Doesn't matter to me as long as it's covering all cases 👍🏼